src/Entity/CrudQueue.php

```diff
@@ -223,6 +258,40 @@ public function sendItem() {
       }
       $created = TRUE;
     }
+    catch (\Aws\Sqs\Exception\SqsException $e) {
```

Comment:
Nice catch (see what I did there? LOL). But seriously, good use of `SqsException`.